internal/cmd/gocdk/build.go

```diff
@@ -80,10 +83,17 @@ func listBuilds(ctx context.Context, pctx *processContext) error {
 	if err != nil {
 		return xerrors.Errorf("list builds: %w", err)
 	}
-	c := pctx.NewCommand(ctx, "", "docker", "images", imageName)
-	if err := c.Run(); err != nil {
+	images, err := docker.New(pctx.env).ListImages(ctx, imageName)
+	if err != nil {
 		return xerrors.Errorf("list builds: %w", err)
 	}
+	for _, image := range images {
+		if image.Repository == "" || image.Tag == "" {
```

Comment:
Just for my understanding, this is reformatting the output, right? Before we just dumped whatever `docker images` outputted, now we are parsing that and then reformatting? Is that necessary/desirable?

Reply:
Correct. Parsing this output is necessary for some of the other logic we have been discussing for the future (more reliable Cloud Run pushes, for instance). I don't really want to have two code paths for listing Docker images, so I figured that I'd make the more structured one here.

Reply:
SGTM.